packages/web-api/src/types/request/chat.ts

```diff
@@ -191,7 +193,7 @@ export type ChatPostMessageArguments = TokenOverridable &
   Authorship &
   Parse &
   LinkNames &
-  Metadata &
+  EntityAndEventMessageMetadata &
```

Comment:
Updated chat.postMessage input; it now accepts both event and entity data (both can be supplied, or one or the other)